### PR TITLE
[MIG][9.0] medical manufacturer

### PR DESCRIPTION
**Pull request by woodbrettm**
* Migration of medical_manufacturer to v9 as it's required for medical_medicament_us.

@lasley ping

### CODE REVIEW

File: medical_manufacturer/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
+
+Known issues / Roadmap
+======================
+
+* Break this into individual modules once everything is settled
```
Comment (woodbrettm) [inline medical_manufacturer/README.rst]: @lasley Is this still relevant?

Comment (lasley) [reply medical_manufacturer/README.rst]: Nope - modules have been broken.. into!?

File: medical_manufacturer/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
+
+Known issues / Roadmap
+======================
+
```
Comment (lasley) [inline medical_manufacturer/README.rst:26]: Add into RoadMap:
`* Integrate manufacturers with OCA/product-attribute/product_brand`

File: medical_manufacturer/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Manufacturer",
+    "summary": "Extends companies with concept of medical manufacturers.",
```
Comment (lasley) [inline medical_manufacturer/__openerp__.py]: This doesn't extend companies. A `res.company` in Odoo is a very different concept from a `res.partner` that is tagged as a company. 

Probably `Extension of medical`, like the readme says

**Review by lasley — APPROVED** [review]
Few notes but otherwise LGTM thanks @bmw95 

@tedsalmon please review

File: medical_manufacturer/__openerp__.py (hunk 3 above)
Comment (woodbrettm) [reply medical_manufacturer/__openerp__.py]: True yeah, my bad.

File: medical_manufacturer/README.rst (hunk 1 above)
Comment (woodbrettm) [reply medical_manufacturer/README.rst]: lol no clue.

Comment (woodbrettm) [reply medical_manufacturer/README.rst]: Perhaps it was done awhile ago but the comment was never removed.

File: medical_manufacturer/__openerp__.py (hunk 3 above)
Comment (woodbrettm) [reply medical_manufacturer/__openerp__.py]: Oh interesting, didn't realize they were so different.

**Comment by woodbrettm** [comment]
Updated!

**Review by tedsalmon — APPROVED** [review]
👍  LGTM!

**Pull request merged**